views/movies/edit.hbs

```diff
@@ -0,0 +1,18 @@
+<h2>edit Movie</h2>
+<form action="/movies/{{movie._id}}/edit" method="POST">
+  <label for="title">Name: </label>
+  <input type="text" name="title" id="title" value="{{movie.title}}">
+
+  <label for="genre">Genre: </label>
+  <input type="text" name="genre" id="genre" value="{{movie.genre}}">
+
+  <label for="plot">Plot: </label>
+  <input type="text" name="plot" id="plot" value="{{movie.plot}}">
+
+  <select name="cast" multiple>
+    {{#each celebrites}}
+    {{{selector ../movie}}}
+    {{/each}}
```

Comment:
Here you can replace this with {{{ options }}} if you prefer to build the selected actors in the backend. If you prefer to use the helper function, you keep these lines unchanged.